web/package/agama-web-ui.spec

```diff
@@ -45,9 +45,9 @@ local-npm-registry %{_sourcedir} install --with=dev --legacy-peer-deps || ( find
 NODE_ENV="production" npm run build
 
 %install
-install -D -m 0644 --target-directory=%{buildroot}%{_datadir}/agama/web_ui %{_builddir}/agama/dist/*.{gz,html,js,map,svg}
+install -D -m 0644 --target-directory=%{buildroot}%{_datadir}/agama/web_ui %{_builddir}/agama/dist/*.{css,gz,html,js,json,map,svg}
 install -D -m 0644 --target-directory=%{buildroot}%{_datadir}/agama/web_ui/fonts %{_builddir}/agama/dist/fonts/*.ttf
-install -D -m 0644 --target-directory=%{buildroot}%{_datadir}/agama/web_ui/assets/logos %{_builddir}/agama/src/assets/products/*.svg
+install -D -m 0644 --target-directory=%{buildroot}%{_datadir}/agama/web_ui/assets/logos %{_builddir}/agama/dist/assets/logos/*.svg
```

Comment:
NP: in the future we might want to go with a `web/install.sh` approach like the other parts of Agama do (part of #2103). I forgot to do it because I was not doing any changes in the front end at the time

Reply:
Yes, would be nice to unify this to have a single place for changes.